src/switcher-ui.tsx

```
@@ -3384,26 +3626,134 @@ function SwitcherApp() {
+                        const hasContext = !!(hit.before || hit.after);
+                        if (!(dupFirst || dupLast) || hits.length > 1 || expanded) {
+                          lines.push(
+                            <div key="hit" style={lineStyle}>
+                              <span style={SNIPPET_LINE_STYLE}>
+                                <span style={SNIPPET_MARKER_STYLE}>match #{hit.promptIndex + 1}</span>
+                                {hits.length > 1 && (
+                                  <span style={{ color: '#777', fontSize: '10px' }} onClick={stop} onMouseDown={stop}>
+                                    {' '}
+                                    <span
+                                      role="button"
+                                      tabIndex={-1}
```

ESLint

[error] Replace `······································` with `············································`

(prettier/prettier)